mcs/class/referencesource/mscorlib/system/runtime/interopservices/ucomitypeinfo.cs

```diff
@@ -212,6 +212,7 @@ public struct DISPPARAMS
         public int cNamedArgs;
     }
 
+#if !FULL_AOT_RUNTIME
     [Obsolete("Use System.Runtime.InteropServices.ComTypes.EXCEPINFO instead. http://go.microsoft.com/fwlink/?linkid=14202", false)]
```

Comment:
why is this and the below excluded on FULL_AOT_RUNTIME ? The `System.Runtime.InteropServices.ComTypes.EXCEPINFO` where the obsolete message points to is included and looks exactly the same

Seems weird to make those two special when the other types in this file are included.